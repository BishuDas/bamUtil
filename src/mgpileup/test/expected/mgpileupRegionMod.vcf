#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	-
1	75	.	A	.	.	.	.	N:BASE:MAPQ:BASEQ:STRAND:CYCLE:GL:TMP	1:A:0:26:F:1:0,3,3,3,30,30,30,30,30,30:tmp
1	78	.	T	.	.	.	.	N:BASE:MAPQ:BASEQ:STRAND:CYCLE:GL	1:T:0:29:F:4:33,33,33,3,33,33,3,33,3,0
1	79	.	N	.	.	.	.	N:BASE:MAPQ:BASEQ:STRAND:CYCLE:GL:TMP	1:N:0:29:F:5:0,0,0,0,0,0,0,0,0,0:tmp
2	75	.	N	.	.	.	.	N:BASE:MAPQ:BASEQ:STRAND:CYCLE:GL:TMP	1:A:0:32:F:1:0,3,3,3,36,36,36,36,36,36:tmp
2	76	.	N	.	.	.	.	N:BASE:MAPQ:BASEQ:STRAND:CYCLE:GL:TMP	1:C:0:32:F:2:36,3,36,36,0,3,3,36,36,36:tmp
2	1751	.	N	.	.	.	.	N:BASE:MAPQ:BASEQ:STRAND:CYCLE:GL	1:A:0:26:F:4:0,3,3,3,30,30,30,30,30,30
2	1754	.	N	.	.	.	.	N:BASE:MAPQ:BASEQ:STRAND:CYCLE:GL:TMP	1:T:0:29:F:7:33,33,33,3,33,33,3,33,3,0:tmp
2	1755	.	N	.	.	.	.	N:BASE:MAPQ:BASEQ:STRAND:CYCLE:GL	1:N:0:29:F:8:0,0,0,0,0,0,0,0,0,0
3	75	.	N	.	.	.	.	N:BASE:MAPQ:BASEQ:STRAND:CYCLE:GL:TMP	1:A:0:26:F:4:0,3,3,3,30,30,30,30,30,30:tmp
3	76	.	N	.	.	.	.	N:BASE:MAPQ:BASEQ:STRAND:CYCLE:GL	1:C:0:29:F:5:33,3,33,33,0,3,3,33,33,33
3	77	.	N	.	.	.	.	N:BASE:MAPQ:BASEQ:STRAND:CYCLE:GL:TMP	1:G:0:29:F:6:33,33,3,33,33,3,33,0,3,33:tmp
3	78	.	N	.	.	.	.	N:BASE:MAPQ:BASEQ:STRAND:CYCLE:GL:TMP	1:T:0:29:F:7:33,33,33,3,33,33,3,33,3,0:tmp
3	79	.	N	.	.	.	.	N:BASE:MAPQ:BASEQ:STRAND:CYCLE:GL	1:N:0:29:F:8:0,0,0,0,0,0,0,0,0,0
