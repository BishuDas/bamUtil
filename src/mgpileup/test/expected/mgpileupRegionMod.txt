Running mgpileup version 1.0
bam file                : testFiles/shortSorted.bam
reference sequence file : ../../bam/test/testFiles/chr1_partial.fa
input VCF file          : testFiles/regionMod.vcf
previous pileup VCF     : testFiles/prevPileup.vcf
output VCF file         : STDOUT
add deletions as bases  : no
